pkg/compose/create.go

```diff
@@ -413,7 +414,7 @@ func (s *composeService) prepareLabels(labels types.Labels, service types.Servic
 
 	var dependencies []string
 	for s, d := range service.DependsOn {
-		dependencies = append(dependencies, fmt.Sprintf("%s:%s:%t", s, d.Condition, d.Restart))
+		dependencies = append(dependencies, fmt.Sprintf("%s:%s:%t:%t", s, d.Condition, d.Restart, d.Required))
```

Comment:
not a fan extending syntax here. I'd prefer we drop dependency if we can't find it.